src/extension.ts

```diff
@@ -99,6 +105,17 @@ export function activate(context: ExtensionContext) {
 
 export function deactivate() { }
 
+function executeAmebaOnWorkspace(ameba: Ameba | null) {
+    if (!ameba) return;
+
+    workspace.textDocuments.forEach(doc => {
+        if (isValidCrystalDocument(doc)) {
+            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
```

Comment:
I'd change the `Init` prefix to sth more apt.

```suggestion
            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
```